tests/feed/test_xml.py

```diff
@@ -26,7 +26,7 @@ def tag(name):
 
 def find_child(parsed, tag_name, is_openmensa_tag=True):
     tag_name = tag_name if not is_openmensa_tag else tag(tag_name)
-    element = parsed.find("./{0}".format(tag_name))
+    element = parsed.find("./{}".format(tag_name))
     if element is None:
         raise KeyError("Element named {0!r} not found.".format(tag_name))
```

Comment:
For consistency I would suggest changing this to `"Element named {!r} not found."...` as well.

Reply:
Updated!